Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -334,6 +432,50 @@ def get_events_alert_type(client: Client, start_date: str, max_fetch: int, last_
     return events, new_last_run_without_next_page
 
 
+def get_events_computer_type(client: Client, start_date: str, max_fetch: int, last_run: dict) -> tuple:
```

Comment:
`tuple` is too broad
```suggestion
def get_events_computer_type(client: Client, start_date: str, max_fetch: int, last_run: dict) -> tuple[list[dict], dict]:
```